internal/core/application/wallet/service.go

```diff
@@ -0,0 +1,374 @@
+package wallet
```

Comment:
If i am not wrong your intention/motivation was to make trade/operator service more readable/maintainable by splitting it to more files and adding folders, I like that but i think we could have keep all it in same folder and not introduce new packages like operator, pubsub, trade, wallet(we should have one pkg for application layer 'application)
After detaching wallet our methods are no longer so long and could have been kept in one file or atleast in same folder and if needed some method could be separated in another file eg. operator_service_fee_fragmenter_split_fund.go for FeeFragmenterSplitFunds.

Reply:
Consider that all app services' factories and interfaces are in `application` package, so the upper layers aren't actually aware of their implementations, and all these "internal" services become just implementation details, like an implementation of the repo manager is for an app service to make a comparison.